simapp/simd/cmd/commands.go

```diff
@@ -50,14 +52,29 @@ func initRootCmd(
 		snapshot.Cmd(newApp),
 	)
 
-	server.AddCommands(rootCmd, newApp, func(startCmd *cobra.Command) {})
+	// server.AddCommands(rootCmd, newApp, func(startCmd *cobra.Command) {})
+	// server v2 commands
+	home, _ := rootCmd.Flags().GetString(flags.FlagHome)
+	if home == "" {
+		home = simapp.DefaultNodeHome
+	}
+	serverv2Cmds, err := serverv2.Commands(
+		home,
+		mockmoduleserver.NewServer("foo"),
+		mockmoduleserver.NewServer("bar"),
+	)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
we should probably remove this to avoid confusion. also what is a server here? why would an app have two?